scripts/update-verification-metadata

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+cd ../
```

Comment:
```suggestion

```

script is supposed to be executed in the project's root directory, so no need to `cd`